CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ Since v0.10.0 this project adheres to [Semantic Versioning](http://semver.org/)
 
 ### Unreleased version
 
+### Version 0.13.0
```

Comment:
How about we add the release dates to these headings a well? We already keep the latest one in the README, but it'd save some digging if one wanted to look into the history of the project.

Reply:
That is actually part of the keepachangelog guidelines, so yes, I'll do that.

Reply:
Done in e2c026c